### ISSUE TITLE
Date Created Sidebar Facet Does Not Function Expected

### ISSUE THREAD

**Issue by aseyedia** [DSpace/dspace-angular#3321]
## Describe the bug
We are using a custom Discovery Search Filter based on `dc.date.created`:

```xml
    <bean id="searchFilterCreated" class="org.dspace.discovery.configuration.DiscoverySearchFilterFacet">
        <property name="indexFieldName" value="dateCreated"/>
        <property name="metadataFields">
            <list>
                <value>dc.date.created</value>
            </list>
        </property>
        <property name="type" value="date"/>
        <property name="facetLimit" value="5"/>
        <property name="sortOrderSidebar" value="COUNT"/>
        <property name="sortOrderFilterPage" value="COUNT"/>
        <property name="isOpenByDefault" value="false"/>
        <property name="pageSize" value="10"/>
        <property name="exposeMinAndMaxValue" value="true"/>
    </bean>
```

On the Search page, this facet appears:

<img width="278" alt="image" src="https://github.com/user-attachments/assets/7fb43136-e30a-4f69-8c27-577a0e5e592b">

1. We would like to set the minimum year to something other than 1950 but it's not clear how.
2. **If we change the year text, for example, if we change 1950 to basically any other year but let's just say 2020, and then hit enter, the page will just start loading and never stop. The URL doesn't change either. It just loads, permanently.** (This is the main issue we are facing).
3. We would like more control over the binning of the years, since our entries do not date before 2015.
4. The slider does not wait until the user releases control of the mouse to enter the query; it immediately searches once the slider stops. This makes precision control more difficult.
5. The slider acts finnicky when trying to select 1-year long intervals i.e. 2022-2023, for example.

## To Reproduce
1. Set a `dc.date.created` Search Filter Facet
2. Try to set the years inside of the year fields and press enter

## Expected behavior

Pressing enter would return the search-filtered query.

**Comment by aseyedia** [comment]
https://github.com/DSpace/dspace-angular/blob/16943fef16a9c219554d26a36b046797d7005cfe/src/app/shared/search/search-filters/search-filter/search-range-filter/search-range-filter.component.ts#L66

To fix the issue of not being able to set the minimum year, I went ahead and changed this to 2015.

**Comment by tdonohue** [comment]
I believe this is a duplicate of #1723, which is waiting on a volunteer.  We all agree the default minimum date should somehow be more configurable or dynamic.

**Comment by aseyedia** [comment]
Looks like the issue with "clicking away" was already addressed in the latest commit for the descendant of this component here with the removal of the `blur` parameter:

https://github.com/DSpace/dspace-angular/commit/56ba23fc3d6b51ed078ffa587f25503b457c97e5#diff-66188afad0c5a4217693eb10cbef3efd0df661c7a32921a6b1e2b60ecf7c7e8bL11-L23

Curious that they kept `[dsDebounce]="250" (change)="onSliderChange($event)"`:

https://github.com/DSpace/dspace-angular/commit/56ba23fc3d6b51ed078ffa587f25503b457c97e5#diff-66188afad0c5a4217693eb10cbef3efd0df661c7a32921a6b1e2b60ecf7c7e8bR36-R37

**Comment by aseyedia** [comment]
I think this issue is more-or-less resolved. I will keep it open for future review but I feel like I was able to find a solution to my problem(s).

**Comment by tdonohue** [comment]
@aseyedia : If this ticket is resolved, then does that mean your PR #3323 was unnecessary?  I'm a little confused as to whether you've found that these issues were fixed by #3236 (the recent work you referenced above), or if you still found new bugs?

Ideally, if this ticket is "resolved" we should close it and its PR.   It can always be reopened later if we find a reason to do so.

**Comment by aseyedia** [comment]
@tdonohue Apologies, allow me to clarify:

In this issue, I (maybe confusingly) outlined a number of small and related issues I had around the "date sidebar search facet."

When I posted https://github.com/DSpace/dspace-angular/issues/3321#issuecomment-2344625288, I was under the impression that #3236 fixed the "clicking away" issue but not the "slider prematurely submitting query" issue. Now I am realizing that even the issue with the slider behavior was resolved. #3236 resolved all of my issues and we can close both this ticket and the PR. Thank you for checking with me.
